README.md

```diff
@@ -1 +1,65 @@
-# community
\ No newline at end of file
+# ROS 2 Security Working Group
+The Security Working Group's mission is to advocate for and implement security features within ROS 2.  The working group negotiates a balance between the benefits of security and the enablement of technology, striving for a secure-by-default design which can be customized to suit a targeted security profile.
+
+This document outlines the governance of the ROS 2 Security Working Group.  Updates to this document will be handled in the same manner as other project updates governed by the Working Group.
+
+## Subprojects
+A significant portion of the Security Working Group's mission is achieved by maintaining ROS subprojects.  The following subprojects are owned by this Working Group:
+ * _TBD:  list project names and team leads_
+
+### Out of Scope
+Although the following ROS subprojects have links to security, they are not governed by the Security Working Group:
+ * _TBD_
+
+### Adding and Maintaining Subprojects
+Subprojects must meet the following criteria:
+* An Approver can add or create a new subproject that will be maintained by this working group.
+* A Member should introduce new subprojects at the next Security Working Group meeting
+* Builds must pass against ROS2 master
+* The ROS2 standard linter set must be enabled and adhered to
+* Builds must have 0 warnings
+* Quality builds must be green (address sanitizer, thread sanitizer, clang thread safety analysis)
+* Test suite must pass
+* Reasonable* code coverage must be met by test suites
+* Issues must be responded to promptly*
+* Releases must go out regularly* when bugfixes or new features are introduced
+
+_*-Precise definition of term not determined_
+
+## Vulnerability Handling
```

Comment:
I think it's a little early for this, yet. We still need to come up with the disclosure policy, then we need to have a conversation with OR about our internal process. Let's remove this section for now to reflect the current status.

Reply:
While the rules are not yet fleshed out, it seems that this is one of the main missions of the working group. I would be in favor of keeping this section with a note "TBD of "Subject to changes" (similar to the list of project) rather than not having it mentionned at all in the readme

Reply:
I actually disagree. The primary responsibility of this WG is to drive the development of security features and best practices in ROS 2, not handle vuln reports. I honestly don't think the security WG should be handling this stuff, or we won't be able to leave membership nearly as open as I think we want it. It should actually be a separate group of engineers, both from the WG and from OR, and the mailing list should be maintained by OR, not this WG as a whole.

Reply:
Fair enough, I think there was some misunderstanding from my side when reading the PR description

> The two primary jobs of the WG are to maintain subprojects and to handle all ROS vulnerabilities (both ROS1 and ROS2).

I now see that this PR clearly states a different mission statement (ROS 2 focused and not mentioning vulnerabilities):

> The Security Working Group's mission is to advocate for and implement security features within ROS 2. The working group negotiates a balance between the benefits of security and the enablement of technology, striving for a secure-by-default design which can be customized to suit a targeted security profile.

In which case no need to include the paragraph about vulnerability handling in here :+1: (and fully ROS 2 focused)

> Long story short, I think we need to talk to OR before we propose or assume anything here.

Yeah it may be beneficial to clarify with OR and working group members what the stance is on security vulnerabilities before advertising this as it may impact both user expectations (as a user I may assume that the security group/team is in charge of security patches) and membership interest